03-task-manager/starter/app.js

```diff
@@ -1 +1,31 @@
-console.log('Task Manager App')
+const express = require('express');
+const app = express();
+
+//set up middleware
+const tasks = require('./routes/task');
+app.use(express.json())
+const connectDB = require('./conectdb/connect');
+require('dotenv').config()
+//routes
+app.get('/hello', (req,res)=> {
+    res.send('TASK MANAGER APP')
+})
+
+app.use('/api/v1/tasks',tasks)
+
+//set up server
+const port = 3000;
+
+//function: connect to database then start server
+const start = async () => {
+    try{
```

Comment:
I am not sure if I have mentioned this before, but I would recommend using a linter for your code so that it formats it and is easier to read.

Reply:
Yes you have but in my vsc its not working.